app/controllers/bars_controller.rb

```diff
@@ -2,7 +2,12 @@ class BarsController < ApplicationController
   before_action :require_login, only: [:new, :edit, :update, :destroy]
 
   def index
-    @bars = Bar.all.order(created_at: :desc).page(params[:page])
+    if params[:search]
+      @bars = Bar.search(params[:search]).order(created_at: :desc).
+                  page(params[:page])
```

Comment:
Use 2 (not 12) spaces for indenting an expression in an assignment spanning multiple lines.